instrumentation/opentelemetry-instrumentation-system-metrics/tests/test_system_metrics.py

```diff
@@ -779,6 +779,22 @@ def test_runtime_get_count(self, mock_gc_get_count):
             _SystemMetricsResult({"count": "1"}, 2),
             _SystemMetricsResult({"count": "2"}, 3),
         ]
-        self._test_metrics(
-            f"process.runtime.{self.implementation}.gc_count", expected
-        )
+        self._test_metrics(f"runtime.{self.implementation}.gc_count", expected)
+
+    # @mock.patch("gc.hooks")
```

Comment:
Please don't add code that is commented out.

Reply:
@ocelotl : Can you please help us on what all other changes need to be done??

Reply:
@ocelotl : Only one of the checks is failing because of below:
ERROR:   pypy3-test-instrumentation-fastapi: commands failed
ERROR:   pypy3-test-instrumentation-starlette: commands failed

Above error is might be because of the Assertion error during the build
>                       self.assertAlmostEqual(duration, point.sum, delta=30)
E                       AssertionError: 35 != 3 within 30 delta (32 difference)

Do need to fix the both one also??
Can you please guide me on this?? It will be really helpful.